T/Top.json

```diff
@@ -0,0 +1,12 @@
+{
+    "word": "Top",
+    "definitions": [
+        "the highest place or part",
+        "any piece of light clothing worn on the part of the body above the waist",
+        "the cover or lid used to close a container or pen",
+        "the most important position",
+        "the leaves of a vegetable plant grown mainly for its roots",
```

Comment:
Hey

Remove ```,``` after the fifth definition in Top.json and update the file.

Thank you for contributing!

Reply:
Done!! Should have checked before hand